README.md

```diff
@@ -6,20 +6,73 @@ This is a Kubernetes Operator (https://coreos.com/operators/) which deploys Mong
 
 This codebase is currently _pre-alpha_, and is not ready for use.
 
-If you are a MongoDB Enterprise customer, or need Enterprise features such as Backup, you may be looking for the [MongoDB Enterprise Operator for Kubernetes](https://github.com/mongodb/mongodb-enterprise-kubernetes).
+If you are a MongoDB Enterprise customer, or need Enterprise features such as Backup, you can use the [MongoDB Enterprise Operator for Kubernetes](https://github.com/mongodb/mongodb-enterprise-kubernetes).
 
+## Installation
 
+### Prerequisites
 
+Before you install the MongoDB Community Kubernetes Operator, you must:
 
-## Licensing
+1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+2. Install [kind](https://kind.sigs.k8s.io/)
+3. Clone this repository.
+   ```
+   git clone https://github.com/mongodb/mongodb-kubernetes-operator.git
+   ```
 
-The source code of this Operator is available under the Apache v2 license.
+### Installing the MongoDB Community Kubernetes Operator
 
-The MongoDB Agent binary in the agent/ directory may be used under the "Free for Commercial Use" license found in agent/LICENSE.
+The MongoDB Community Kubernetes Operator is a collection of [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) and a controller.
+
+To install the MongoDB Community Kubernetes Operator:
+
+1. Change to the directory in which you cloned the repository.
+2. Install the [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+   a. Invoke the following `kubectl` command:
+      ```
+      kubectl create -f deploy/crds/mongodb.com_mongodb_crd.yaml
+      ```
+   b. Verify that the Custom Resource Definitions installed successfully:
+      ```
+      kubectl get crd/mongodb.mongodb.com
+      ```
+3. Install the Operator.
+   a. Invoke the following `kubectl` command to install the Operator in the `default` namespace:
```

Comment:
this command isn't always guaranteed to create the operator in the `default` namespace. This will be the case if no namespace is specified in the operator.yaml or one wasn't provided with the kubectl command. And the current context does not have a default namespace specified.

I think we should indicate that you can specify a namespace 
`kubectl create -f deploy --namespace <some-namespace>`

Reply:
Instead of specifying the `--namespace <..>` we could say that if you want to use a namespace different than `default` you should specify it on each of the `kubectl` invokations.

Reply:
I see what Cian is saying, because they could have another namespace set in the context, so if they use this it wouldn't necessarily be `default`. I think its safest just to specify the namespace in every command.

Reply:
I think specifying `--namespace` is a good idea. I'm not inclined to set the default namespace in the context, beause it means chaning the user's configuration globally, making this not easy to integrate with other tools.